app/graphql/types/reference_repository_type.rb

```diff
@@ -0,0 +1,206 @@
+
+# frozen_string_literal: true
+
+class ReferenceRepositoryType < BaseObject
+  description "Information about repositories"
```

Comment:
I was not anticipating a new PUBLIC interface (ReferenceRepositoryType) for this. particularly when it seems, in a first view, exactly the same fucntionality exposed in RepositoryType. I was anticiapating you will just replace it. 

Particularly with public interfaces I think we should avoid "exposing complexity" of the our intricated system. Could we discuss this @jrhoads?

Reply:
Basically, if the GraphpQL user (we are not the only user) needs Repositories they go to a single place, and if they need a subset they filter using the same type.

Reply:
Happy to discuss (and change).  I had some concerns about someone wanting to interact with a "Client-type" repository from something like Fabrica or the like.  I'm not sure if that is a realistic concern or not.

Reply:
We had a conversation on zoom and I will update the GraphQL endpoints so this replaces the Repository endpoint rather than adding a new ReferenceRepository endpoint.